apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
I bet we also have a lot of unused strings.

On Fri, Jan 23, 2015 at 12:30 PM, Brad Buchanan <notifications@github.com>
wrote:

> In apps/i18n/netsim/en_ploc.json
> <https://github.com/code-dot-org/code-dot-org/pull/1133#discussion_r23477988>
> :
>
> > @@ -0,0 +1,3 @@
> > +{
> > +  "yes": "Yes"
>
> We may want to add a task to audit these app localization files, and move
> very common localizations into the common file. "yes": "Yes" shows up in
> the individual localization files for every app. I wonder how many other
> duplicates we've got.
>
> [image: screenshot from 2015-01-23 12 28 27]
> <https://cloud.githubusercontent.com/assets/1615761/5882126/8a0e4250-a2fb-11e4-81cc-0beac8f0615e.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/1133/files#r23477988>.
>